HLTrigger/NGTScouting/python/HLTNanoProducer_cff.py

```diff
@@ -28,6 +29,7 @@
     + hltMuonTable
     + hltPFCandidateTable
     + hltJetTable
+    + hltTauTable
```

Comment:
my personal proposal would be to remove this producer from here for the time being and put it back once we have a more versatile system in place for the regular HLT menu. @artlbv 
We can take care of putting it back once it's possible.

Reply:
Ok, good, so I will remove the taus from the non-scouting sequence for now, thanks!

Reply:
@artlbv FYI, we followed up at https://github.com/cms-sw/cmssw/pull/48425